### PR TITLE
Issue 21 feat/UI prepare home screen

### PR DESCRIPTION
**Pull request by Mariusz-web**

### CODE REVIEW

**Comment by Veryyapeee** [comment]
IMO that should be centered vertically
Screenshot 2022-05-09 at 21 02 37

File: apps/client/src/app/Router.tsx [hunk 1]
```diff
@@ -57,18 +57,15 @@ export const Router = () => {
                         </ProtectedRoute>
                     }
                 >
+                    <Route path={ROUTES.HOME} element={<Home />} />
```
Comment (Veryyapeee) [inline apps/client/src/app/Router.tsx:60]: why did you added home route here, and commented the one above? xd

File: apps/client/src/app/sceens/BaseScreen/BaseScreen.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ export const BaseScreen = () => {
     const isLogged = true; //TODO connect with api(detect if user is logged)
     return (
         <div className='base-screen'>
-            {isLogged ? <Header /> : null}
+            {/* {isLogged ? <Header /> : null} */}
```
Comment (Veryyapeee) [inline apps/client/src/app/sceens/BaseScreen/BaseScreen.tsx:9]: why is it commented?

File: apps/client/src/app/sceens/Home/Home.styles.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
+export const homeStyle = {
```
Comment (Veryyapeee) [inline apps/client/src/app/sceens/Home/Home.styles.ts:1]: we could use emotion in here, which is by default dependency of MUI

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: apps/client/src/app/sceens/BaseScreen/BaseScreen.tsx (hunk 2 above)
Comment (Mariusz-web) [reply apps/client/src/app/sceens/BaseScreen/BaseScreen.tsx:9]: I forgot to change it

**Review by Veryyapeee — APPROVED** [review]
Looks good, remember to uncomment code that you commented for testing purposes

**Pull request merged**